### ISSUE TITLE
The 'simple' micropub option sets link as an "h"  

### ISSUE THREAD

**Issue by davidmead** [aaronpk/OwnYourSwarm#38]
Hi.

[My check in]( http://davidjohnmead.com/blog/2018/07/07/19861/) looks like this...
```
<p>Checked into <a class="h-card p-location" href="h">h</a>.</p>
<div class="e-content">
Checked in at Swenson's
</div>
```
...whereas [my previous check in](http://davidjohnmead.com/blog/2018/06/17/19809/), using the simple method, looked like this

```
<p>Checked into <a class="h-card p-location" href="https://foursquare.com/v/4ad4bff2f964a520d4e920e3">Blossom Music Center</a>.</p>
<div class="e-content">
Imagine Dragons - Josh's first concert!
</div>
```

Seems to be adding the description to the e-content instead of the h-entry. Is this Swarm/Foursquare not passing the info along, or something wrong OwnYourSwarm?  I am also getting duplicate postings, but that's happening in Swarm.

**Comment by aaronpk** [comment]
Yes, this is working as intended.

The JSON mode sends an array, and if the Micropub server is not expecting that you'll see weird stuff like that. 

I'm guessing this is not the latest version of the Micropub plugin?

**Comment by davidmead** [comment]
I have version 1.4.3 of the WP micropub plugin, but I'm seeing this when I use the simple method, **not** the JSON.

![image](https://user-images.githubusercontent.com/1266930/42420734-18ef7406-8298-11e8-9798-e8136c493a36.png)

**Comment by aaronpk** [comment]
I'm confused. In your original comment, the "simple" method example looks fine. It's the first one (which I assumed to be the JSON version) that is broken.

**Comment by aaronpk** [comment]
Did anything about your site change between the two checkins?

I suspect this is actually a problem with your theme or the Post Kinds plugin. Nothing with OwnYourSwarm has changed recently.

**Comment by davidmead** [comment]
I did try using the JSON method for [a checkin with a comment and photo]( http://davidjohnmead.com/blog/2018/06/25/19830/), but it didn't work (no media endpoint) so I swapped back.  That's when I noticed this odd behavior. 

I have version 3.0.9 of PostKinds plugin installed, but not active.

There was a theme update, but I use a child theme for some of the indieweb stuff and I didn't change anything for that.
